lib/MusicBrainz/Server/Controller/Admin.pm

```diff
@@ -21,6 +21,8 @@ sub edit_user : Path('/admin/user/edit') Args(1) RequireAuth HiddenOnSlaves
         unless $c->user->is_account_admin or DBDefs->DB_STAGING_TESTING_FEATURES;
 
     my $user = $c->model('Editor')->get_by_name($user_name);
+    $c->stash->{viewing_own_profile} = $c->user_exists && $c->user->id == $user->id;
+
```

Comment:
To allow admin pages to show all the tabs, including the viewing_own_profile one (this also fixes a bug of sorts where currently the donation check page can't be accessed from the admin edit user interface on your own page - affecting the entirety of 3 or so users).